articles/securing-django-api/index.md

```diff
@@ -0,0 +1,251 @@
+In this tutorial, we will secure our TODO API endpoints that we created in this [article](/engineering-education/django-crud-api/). We will start by implementing Token-based authentication and then implement javascript web tokens (JWT).
+
+### Token-based authentication
+Token-based authentication works by getting a token for the correct username and password used to perform subsequent requests to the server.
+
+**Project setup**
+Add `'rest_framework.authtoken'` to the apps list in `django_todo` settings.py file.
+
+```python
+# ./django_todo/settings.py
+...
+INSTALLED_APPS = [
+    ...
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'todo',
+    'rest_framework',
+    'rest_framework.authtoken'
+]
+```
+Add `'rest_framework.authentication.TokenAuthentication'` to `REST_FRAMEWORK` dictionary in the `django_todo` project `settings.py`.
+
+```python
+REST_FRAMEWORK = {
```

Comment:
Check this section. There is some overlap with the material presented later. In the later section, you mention the same set of steps, but with the addition of coreapi, which is not mentioned here.

Reply:
I have corrected this. The coreapi i explained about it in the previous article.